presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -224,10 +224,10 @@ columnAliases
     ;
 
 relationPrimary
-    : qualifiedName                                                   #tableName
-    | '(' query ')'                                                   #subqueryRelation
-    | UNNEST '(' expression (',' expression)* ')' (WITH ORDINALITY)?  #unnest
-    | '(' relation ')'                                                #parenthesizedRelation
+    : qualifiedName                                                             #tableName
+    | '(' query ')'                                                             #subqueryRelation
+    | (TABLE | UNNEST) '(' expression (',' expression)* ')' (WITH ORDINALITY)?  #unnest
```

Comment:
But does it conflict with anything? I thought it could be a nice extension to standard as it is already implemented for `UNNEST`.

Is there any problem of having this, or is just non standard compliant?